### PR TITLE
Feature/faster any

### PR DESCRIPTION
**Pull request by ObiWahn**
https://github.com/arangodb/planning/issues/1439

### CODE REVIEW

File: arangod/RocksDBEngine/RocksDBIterators.cpp [hunk 1]
```diff
@@ -193,26 +192,53 @@ RocksDBAnyIndexIterator::RocksDBAnyIndexIterator(
   TRI_ASSERT(_iterator);
 
   _total = col->numberDocuments(trx);
-  uint64_t off = RandomGenerator::interval(_total - 1);
+  _forward = RandomGenerator::interval(uint16_t(2)) ? true : false; //prefer forward
```
Comment (jsteemann) [inline arangod/RocksDBEngine/RocksDBIterators.cpp]: This strongly prefers forward iteration, as the generated values will mean the following:
- 0: backward
- 1: forward
- 2: forward

File: arangod/RocksDBEngine/RocksDBIterators.cpp [hunk 2]
```diff
@@ -193,26 +192,53 @@ RocksDBAnyIndexIterator::RocksDBAnyIndexIterator(
...
+      static_cast<RocksDBCollection*>(col->getPhysical())->objectId(),
+      RandomGenerator::interval(UINT64_MAX)
+    );
+    _iterator->Seek(initialKey.string());
+
+    if (checkIter()) {
+      if (_forward) {
+        while (steps-- > 0) {
+          _iterator->Next();
+          if(!checkIter()) break;
```
Comment (jsteemann) [inline arangod/RocksDBEngine/RocksDBIterators.cpp]: violates our coding conventions

File: arangod/RocksDBEngine/RocksDBIterators.cpp [hunk 3]
```diff
@@ -193,26 +192,53 @@ RocksDBAnyIndexIterator::RocksDBAnyIndexIterator(
...
+        }
+      } else {
+        while (steps-- > 0) {
+          _iterator->Prev();
+          if(!checkIter()) break;
```
Comment (jsteemann) [inline arangod/RocksDBEngine/RocksDBIterators.cpp]: too

File: arangod/RocksDBEngine/RocksDBIterators.cpp (hunk 1 above)
Comment (ObiWahn) [reply arangod/RocksDBEngine/RocksDBIterators.cpp]: ok i'll make it totally fair again

**Pull request merged**